resources/metadata/cloudFoundryCreateService.yaml

```diff
@@ -0,0 +1,157 @@
+        aliases:
+          - name: cloudFoundry/servicePlan
+      - name: cfServiceInstanceName
+        type: string
+        description: Parameter for naming the Service Instance when creating Cloud Foundry Service
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/serviceInstanceName
+      - name: cfServiceBroker
+        type: string
+        description: Parameter for Service Broker to be used when creating Cloud Foundry Service
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/serviceBroker
+      - name: cfCreateServiceConfig
+        type: string
+        description: Path to JSON file or JSON in-line String for Cloud Foundry Service creation
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/createServiceConfig
+      - name: cfServiceTags
+        type: string
+        description: Flat list of Tags to be used when creating Cloud Foundry Service in a single string
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/serviceTags
+      - name: serviceManifest
+        type: string
+        description: Path to Cloud Foundry service manifest in YAML format for multiple service creations that are being passed to a Create-Service-Push cf cli plugin
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/serviceManifest
+      - name: manifestVariables
+        type: '[]string'
+        description: Defines a List of variables as key-value Map objects used for variable substitution within the file given by manifest. Defaults to an empty list, if not specified otherwise. This can be used to set variables like it is provided by `cf push --var key=value`. The order of the maps of variables given in the list is relevant in case there are conflicting variable names and values between maps contained within the list. In case of conflicts, the last specified map in the list will win. Though each map entry in the list can contain more than one key-value pair for variable substitution, it is recommended to stick to one entry per map, and rather declare more maps within the list. The reason is that if a map in the list contains more than one key-value entry, and the entries are conflicting, the conflict resolution behavior is undefined (since map entries have no sequence). Variables defined via `manifestVariables` always win over conflicting variables defined via any file given by `manifestVariablesFiles` - no matter what is declared before. This is the same behavior as can be observed when using `cf push --var` in combination with `cf push --vars-file`
```

Comment:
```suggestion
        description: Defines a list of variables as key-value Map objects used for variable substitution within the file given by manifest. Defaults to an empty list, if not specified otherwise. This can be used to set variables like it is provided by `cf push --var key=value`. The order of the maps of variables given in the list is relevant in case there are conflicting variable names and values between maps contained within the list. In case of conflicts, the last specified map in the list will win. Though each map entry in the list can contain more than one key-value pair for variable substitution, it is recommended to stick to one entry per map, and rather declare more maps within the list. The reason is that if a map in the list contains more than one key-value entry, and the entries are conflicting, the conflict resolution behavior is undefined (since map entries have no sequence). Variables defined via `manifestVariables` always win over conflicting variables defined via any file given by `manifestVariablesFiles` - no matter what is declared before. This is the same behavior as can be observed when using `cf push --var` in combination with `cf push --vars-file`
```

Reply:
What is the difference between the suggestion and the current state?